docs/UserGuide.adoc

```diff
@@ -212,15 +218,15 @@ The `redo` command fails as there are no `undo` commands executed previously.
 `redo` (reapplies the `clear` command) +
 // end::undoredo[]
 
-=== Clearing all entries : `clear`
+=== Clearing all entries : `clear` (alias `clr`)
```

Comment:
do we really need an alias for such a command? I think it is not meant to be used frequently

Reply:
I agree. Might be better not to have an alias for this one to minimise accidental clearing